boards/x86_64/qemu_x86_64/qemu_x86_64.yaml

```diff
@@ -6,5 +6,6 @@ arch: x86_64
 toolchain:
   - zephyr
   - host
+  - xtools
```

Comment:
why xtools? This should work with the zephyr variant, no?

Reply:
Updated the patch to use the newly built x86_64 toolchain for when toolchain variant is zephyr. This is for testing only with artifacts from CI. This needs to be updated for new SDK release.

Reply:
Should work for both zephyr and xtools now.